draft-amjad-cfrg-partially-blind-rsa.md

```diff
@@ -206,14 +206,16 @@ Inputs:
 - bits, length in bits of the safe prime
 
 Outputs:
-- p, a safe prime integer
+- p, a safe prime integer > 2^(bits-0.5)
 
 Steps:
 1. p' = random_prime(bits - 1)
 2. p = (2 * p') + 1
-3. if is_prime(p) is True, output p, else go to step 1.
+3. if is_prime(p) is True and p > 2^(bits-0.5), output p, else go to step 1.
```

Comment:
Would it make more sense to run is_prime only if the lower bound on p is satisfied?

Reply:
Makes sense. Done!

Reply:
I misread what you wanted. Re-fixing.